gcalcli

```diff
@@ -1697,10 +1728,17 @@ class gcalcli:
 
         event = {}
         event['summary'] = stringToUnicode(eTitle)
-        event['start'] = {'dateTime': eStart,
-                          'timeZone': self.cals[0]['timeZone']}
-        event['end'] = {'dateTime': eEnd,
-                        'timeZone': self.cals[0]['timeZone']}
+
+        if FLAGS.allday:
```

Comment:
The reason I have to add None elsewhere is because the Google Calendar API does not allow you to have both 'dateTime' and 'date' set at the same time.  In this portion of the code, we're adding a new event, so neither was set to begin with, so there's no need to set something to None (though we could certainly do it and it would cause no problem).

Elsewhere (in the EditEvent section), the None becomes important in case you are toggling between partial- and full-day events.

So, when you say 'I would remove those lines to reduce length', if you are referring to the EditEvent section, you cannot remove them.  I tried it first without them, and got errors which led to finding out how the API worked in that situation.